src/main.cpp

```diff
@@ -164,25 +164,21 @@
 
 void IRAM_ATTR ScreenUpdateLoopEntry(void *);
 
-// Global support objects and global values
+//
+// Global Variables
+//
+
 DRAM_ATTR std::unique_ptr<SystemContainer> g_ptrSystem;
+DRAM_ATTR std::unique_ptr<EffectFactories> g_ptrEffectFactories = nullptr;

```

Comment:
I intentionally didn't include g_ptrEffectFactories in the main.cpp globals. In my mind it's a global that's local to network.cpp - which has effectmanager.h as one of it's two primary defines. I suggest we leave it in network.cpp.

Reply:
Suggested change here:

```suggestion
```